### ISSUE TITLE
decimalPlaces option 

### ISSUE THREAD

**Issue by chalioui** [JohnCoene/countup#1]
The option decimalPlaces don't work in the countup function :
![image](https://user-images.githubusercontent.com/52160761/67618995-89f41100-f7f6-11e9-8ee6-e73dd692c8ae.png)

**Comment by JohnCoene** [comment]
Yes indeed. This is because the package runs on a previous version of the JavaScript library. I did not manage to update it due to the requirement for the `type=module` in the script tag sourcing Countup.js, [see README](https://github.com/inorganik/CountUp.js#example-with-vanilla-js).

**Comment by mawtee** [comment]
@JohnCoene Any sense whether, given the passing of time, this issue may not be fixable? Thanks for putting in the work to create this package btw!

**Comment by JohnCoene** [comment]
Updated to latest version, it should work now, note that the API has slightly changed, accepting `...` instead of options `list`.

EDIT: see README for example

**Comment by mawtee** [comment]
Thank you @JohnCoene - this is very helpful!
